google_takeout_parser/parse_json.py

```diff
@@ -125,12 +125,12 @@ def _parse_app_installs(p: Path) -> Iterator[Res[PlayStoreAppInstall]]:
     for japp in json_data:
         try:
             yield PlayStoreAppInstall(
-                title=japp["install"]["doc"]["title"],
-                deviceName=japp["install"]["deviceAttribute"].get("deviceDisplayName"),
-                deviceCarrier=japp["install"]["deviceAttribute"]["carrier"],
-                deviceManufacturer=japp["install"]["deviceAttribute"]["manufacturer"],
-                lastUpdateTime=parse_json_utc_date(japp["install"]["lastUpdateTime"]),
-                firstInstallationTime=parse_json_utc_date(japp['install']['firstInstallationTime']),
+                title=japp.get("install", {}).get("doc", {}).get("title"),
```

Comment:
yeah, this seems okay. it might actually be nicer to have a bit more code here so when google eventually changes things we get a good error message?

so maybe like a 

```
install = japp.get("install")
if install is None:
    yield RuntimeError(f"Could not find install info in {str(japp)}")
    continue
```

as long as that attribute is a required one for the current understanding of the model, if the data doesnt have it that should be a breaking change and should warn us, so we can 'fix' it, either updating the key name, checking for some new key etc. while still managing backwards compatibility with old data

Reply:
In general I'm okay with this pattern if its some optional data, liked marked `Optional[str]` on the model. But if its required data, then we shouldn't be using `.get`, or we should be checking if the response `is not None` before yielding an error and then `continue`ing to the next item in `json_data`

Reply:
Thanks for raising, the required fields slipped my mind while making this change. I'll change them back to the previous convention without the `.get()` which is probably a lot less code than having to write `if` statements for every parent and should achieve the same result.